src/models/PenaltyIdentifier.ts

```diff
@@ -2,7 +2,8 @@ import { PenaltyList } from 'ch-sdk-node/dist/services/lfp';
 
 export interface PenaltyIdentifier {
     companyNumber: string;
-    penaltyReference: string;
+    userInputPenaltyReference: string;
+    penaltyReference?: string;
```

Comment:
should this be a required field? We could set this to the value of the `userInputPenaltyReference` so that it's never null

Reply:
It's not that it isn't required, it's that it won't be defined when the object is instantiated

Reply:
There's some some logic behind this which does that after the user has pressed continue in the PenaltyDetailsPage